src/Survey.jl

```diff
@@ -2,6 +2,7 @@ module Survey
 
 using DataFrames
 using Statistics
+import Statistics: quantile
```

Comment:
Doesn't this conflict with the import on line 7?

Reply:
I havent received any error messages, but am still looking into it.